app/assets/stylesheets/base/globals.scss

```diff
@@ -0,0 +1,30 @@
+
+.globalButtons {
```

Comment:
Bootstrap already has global styles defined, I think we should stick to that rather than defining new ones.
Also it's common convention to follow a `hyphenated-naming-convention` for css classes and ids.